quick_start/sample_app.py

```diff
@@ -0,0 +1,382 @@
+    def load_configs(cls, config_file_name: str) -> None:
+        """
+        Loads JSON configuration file, and sets the metadata in place.
+        :param config_file_name: Configuration file path.
+        """
+        try:
+            with open(config_file_name, "r") as config_file:
+                json_dict = json.load(config_file)
+
+        except Exception as ex:
+            Utils.error_handler(f"Couldn't read load config file from file '{config_file_name}'", ex)
+
+        cls.convert_config_json_fields(json_dict, config_file_name)
+
+    @classmethod
+    def convert_config_json_fields(cls, json_dict: dict, config_file_name: str) -> None:
+        """
+        Converts dict object - JSON style camelCase to ConfigJson object - python snake_case
+        :param config_file_name: JSON configuration file.
+        :param json_dict: Dict of JSON configuration - styled in camelCase
+        """
+        cls.config.use_existing_table = json_dict["useExistingTable"]
+        cls.config.database_name = json_dict["databaseName"]
+        cls.config.table_name = json_dict["tableName"]
+        cls.config.table_schema = json_dict["tableSchema"]
+        cls.config.kusto_uri = json_dict["kustoUri"]
+        cls.config.ingest_uri = json_dict["ingestUri"]
+        cls.config.data_to_ingest = cls.convert_config_data_fields(json_dict["data"])
+        cls.config.alter_table = json_dict["alterTable"]
+        cls.config.query_data = json_dict["queryData"]
+        cls.config.ingest_data = json_dict["ingestData"]
+        cls.config.authentication_mode = json_dict["AuthenticationMode"]
+        cls.config.wait_for_user = json_dict["WaitForUser"]
+        cls.config.wait_for_ingest_seconds = json_dict["WaitForIngestSeconds"]
+        cls.config.batching_policy = json_dict["BatchingPolicy"]
+
+        if (
+            cls.config.database_name is None
+            or cls.config.table_name is None
+            or cls.config.table_schema is None
+            or cls.config.kusto_uri is None
+            or cls.config.ingest_uri is None
+            or cls.config.data_to_ingest is None
+            or cls.config.authentication_mode is None
+            or cls.config.wait_for_user is None
+            or cls.config.wait_for_ingest_seconds is None
+        ):
+            Utils.error_handler(f"File '{config_file_name}' is missing required fields")
+
+    @classmethod
+    def convert_config_data_fields(cls, data_dicts: list[dict]) -> list[ConfigData]:
+        """
+        Converts dict object  - JSON style camelCase to ConfigData object - python snake_case
+        :param data_dicts: Dict of data sources list to ingest from - styled in camelCase
+        :return: ConfigData Data sources list to ingest from - styled in snake_case
+        """
+        config_data_list = []
+
+        for data_dict in data_dicts:
+            config_data = ConfigData()
+            config_data.source_type = SourceType[inflection.underscore(data_dict["sourceType"])]
```

Comment:
If you are using inflection here, why not use it for all of the values? it would save you a lot of code

Reply:
You use the infliction library to convert snake_case to camelCase for source_type.

But for all of the keys in data_dict, you manually convert them to snake_case, I'm saying you can use the library the save you this manual conversion.